Mathlib/Data/Int/Log.lean

```diff
@@ -122,6 +122,16 @@ theorem log_zero_right (b : ℕ) : log b (0 : R) = 0 :=
 theorem log_one_right (b : ℕ) : log b (1 : R) = 0 := by
   rw [log_of_one_le_right _ le_rfl, Nat.floor_one, Nat.log_one_right, Int.ofNat_zero]
 
+@[simp]
+theorem log_zero_base (r : R) : log 0 r = 0 := by simp only [log, Nat.log_zero_left, Nat.cast_zero,
```

Comment:
Are there any other lemmas that use `base` in the name? Otherwise `left` is probably better

Reply:
I changed it to `left`